drivers/lora/sx1276.c

```diff
@@ -362,8 +363,61 @@ void SX1276SetRfTxPower(int8_t power)
 	}
 }
 
+static int sx1276_config_valid(const struct lora_modem_config *config)
+{
+	if (config->frequency == 0)
```

Comment:
That’s only part of it. I should check if the pointer is valid (I might have forgotten that), but the main purpose is to check if the configuration is “sane”. There are two reasons for this: First, some chips allow you to load illegal values into their configuration registers. The behaviour in that case is undefined and potentially very undesirable (e.g., noise outside of the configured frequency). Second, since we apply the configuration lazily, any illegal value that is caught won’t be flagged until we send or receive a packet.